web/static/js/components/grouping_stage_idea_card.jsx

```diff
@@ -46,15 +51,21 @@ export class GroupingStageIdeaCard extends Component {
       }
     }
 
-    if (idea.inEditState) {
+    if (idea.ephemeralGroupingId) {
+      const color = randomColor({ seed: idea.ephemeralGroupingId })
```

Comment:
passing `seed: ephemeralGroupingId ` ensures uniqueness of the colors used to identify groups, but this could be improved, as we sometimes get colors in a similar area of the color wheel:

Reply:
![image](https://user-images.githubusercontent.com/4386645/62329473-341c0100-b473-11e9-8c1b-517af625a4fc.png)